src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure.Core/Configurations/ASPNetBenchmark.Configuration.cs

```diff
@@ -11,17 +11,20 @@ public sealed class ASPNetBenchmarksConfiguration : ConfigurationBase
     public sealed class Run : RunBase
     {
         public string? corerun { get; set; }
+        public string? framework_version { get; set; }
     }
 
     public class Environment
     {
         public Dictionary<string, string> environment_variables { get; set; } = new();
         public uint default_max_seconds { get; set; } = 300;
+        public string framework_version { get; set; } = "net8.0";
```

Comment:
Is this default value going to always "win" the choice above?

Reply:
Yes, this was the value hardcoded before in the crank command.

Reply:
I don't mean to question the specific value.  Is line 84

``` c#
    if (!string.IsNullOrEmpty(configuration.Environment.framework_version))
```

referring to this value?  If so, then if it has a default hard-coded value, then won't the `if` always be `true`?  (and also the same string appears on line 82)

Reply:
Ah - true. Will change so that we are only checking if there is an override.